api/v1beta1/agentserviceconfig_types.go

```diff
@@ -50,6 +50,13 @@ type MustGatherImage struct {
 	Url string `json:"url"`
 }
 
+type ImageServiceOptions struct {
+	// InsecureSkipVerify toggles whether or not the image service verifies
+	// certificates when downloading iso images from a server.
+	// +optional
+	InsecureSkipVerify string `json:"insecureSkipVerify"`
```

Comment:
Any reason this shouldn't be a bool?
Otherwise I feel like we should validate what the user puts here.

Reply:
`bool` is [generally discouraged](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types).

Reply:
That's a great link, thanks!